schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED DEFAULT AS IDENTITY,
+    name VARCHAR(50),
+    date_of_birth DATE,
+    escape_attempts CHAR(15),
```

Comment:
- [ ] Kindly change the type of escape_attempts to be `integer` instead of `char(15)` as in the requirement.
![image](https://user-images.githubusercontent.com/24703171/171050162-7c524b40-ca38-4e1e-a429-647c81453279.png)